doc/TOR.md

```diff
@@ -255,48 +265,51 @@ Other nodes will not be able to `connect` to you unless you communicate them how
 You will find your .onion address with the command `lightning-cli getinfo` and the other nodes will 
 be able to connect to it through the 9735 port.
 
-#### Case #6	c-lightning has a public IP address and a fixed TOR V3 service address and a TOR V2 service address
-
-You will be reachable via Clearnet, via TOR to the .onion V3 address and the .onion V2 address if this
-last is communicated to the node that wants to connect with our node.
-
-Once the .onion addresses have been created with the procedures oulined above, 
-to make your external IP address public you add: `--bind-addr=yourInternalAddress:port --announce-addr=yourexternalIPAddress:port`
+#### Case #6 c-lightning has a public IP address and a fixed Tor V3 service address and a Tor V2 service address
 
-To make your external .onion addresses public you add: `--bind-addr=yourInternalIPAddress:port --announce-addr=yourexternalIPAddress:port --announce-addr=.onionAddressV2:port --announce-addr=.onionAddressV3:port`
+You will be reachable via Clearnet, via Tor to the .onion V3 address and the 
+.onion V2 address if this last is communicated to the node that wants to 
+connect with our node.
 
+to make your external IP address public you add: 
+`--bind-addr=yourInternalAddress:port --announce-addr=yourexternalIPAddress:port`. 
+If the node is not on an internal network the option will be:
+`--addr=yourexternalIPAddress:port`.
 
-#### Case #7	c-lightning has no public IP address and a fixed TOR V3 service address and fixed TOR V2 service  address a 3rd non persisten V2 address
+Once the .onion addresses have been created with the procedures [oulined above](#creation-of-an-hidden-service-for-a-persistent-.onion-address), 
+the node is already reachable at the .onion address.
 
-External users can connect to this node by TOR V2 and V3 and a random V2 until next tor release, then also (V3 randomly).
+To make your external .onion addresses public you add: `--announce-addr=.onionAddressV2:port --announce-addr=.onionAddressV3:port` to the options to publish your IP number.
 
-The Persistent addresses can be created with the steps outlined above.
+#### Case #7 c-lightning has no public IP address and a fixed Tor V3 service address and fixed Tor V2 service  address a 3rd non persisten V2 address
```

Comment:
s/  address a 3rd non persisten V2 address/ and also a non-persistent V2 Tor address/?